src/components/HistoryDialog/index.tsx

```diff
@@ -51,149 +49,236 @@ const StyledTitle = styled("p")({
-    right: "50%",
-    position: "absolute",
-    transform: "translateX(50%)",
-    zIndex: "1",
+const StyledIcon = styled("div")({
+  position: "absolute",
+  top: "50%",
+  transform: "translateY(-50%)",
+  lineHeight: "0",
+  "& img": {
+    WebkitUserDrag: "none",
   },
-  // Add vertical lines between timeline item dots (bottom)
-  "&:not(:last-of-type) .MuiTimelineSeparator-root::after": {
-    content: '" "',
-    width: "6px",
-    background: "#fff",
-    top: "50%",
-    bottom: "0",
-    right: "50%",
-    position: "absolute",
-    transform: "translateX(50%)",
-    zIndex: "1",
+});
+
+const StyledCloseButton = styled(Button)({
+  minWidth: "137px",
+  fontSize: "16px",
+  fontWeight: "700",
+  borderRadius: "8px",
+  textTransform: "none",
+  color: "#fff",
+  borderColor: "#fff",
+  margin: "0 auto",
+  "&:hover": {
+    borderColor: "#fff",
   },
 });
 
-const StyledTimelineSeparator = styled(TimelineSeparator)({
+const StyledHeaderRow = styled(Grid)({
+  borderBottom: "0.5px solid #375F9A",
+  paddingBottom: "8px",
+  marginBottom: "4px",
+});
+
+const StyledHeaderItem = styled(Typography)<React.CSSProperties>((styles) => ({
+  fontFamily: "Public Sans",
+  fontWeight: "300",
+  fontSize: "8px",
+  textAlign: "center",
+  textTransform: "uppercase",
+  color: "#9FB3D1",
+  userSelect: "none",
+  ...styles,
+}));
+
+const HeaderGap = styled("div")({
+  height: "40px",
+});
+
+const StyledEventRow = styled(Grid)({
+  padding: "17px 0",
+  borderBottom: "0.5px solid #375F9A",
+  alignItems: "center",
+});
+
+const BaseItemTypographyStyles: React.CSSProperties = {
+  fontFamily: "Public Sans",
+  fontWeight: "400",
+  fontSize: "13px",
+  letterSpacing: "0.0025em",
+  userSelect: "none",
+};
+
+const StyledEventItem = styled(Typography)<React.CSSProperties>(
+  ({ textAlign = "center", color = "inherit" }) => ({
+    ...BaseItemTypographyStyles,
+    textAlign,
+    color,
+  })
+);
+
+const DotContainer = styled("div")({
   position: "relative",
-  minHeight: "70px",
+  width: "100%",
+  height: "100%",
 });
 
-const StyledTimelineDot = styled(TimelineDot)({
-  background: "#fff",
-  borderWidth: "4px",
-  margin: "0",
-  zIndex: "2",
+const VerticalDot = styled("div")({
   position: "absolute",
   top: "50%",
-  left: "50%",
-  transform: "translate(-50%, -50%)",
+  left: "0px",
+  transform: "translateY(-50%)",
+  content: '""',
+  width: "16px",
+  height: "16px",
+  borderRadius: "50%",
+  background: "white",
 });
 
-const StyledTimelineVerticalLine = styled("span")({
-  width: "60px",
-  height: "2px",
-  background: "#fff",
+const TopConnector = styled("div")({
+  content: '""',
+  position: "absolute",
+  left: "5px",
+  bottom: "0",
+  width: "6px",
+  height: "27px",
+  background: "white",
+});
+
+const BottomConnector = styled("div")({
+  content: '""',
+  position: "absolute",
+  left: "5px",
+  top: "0",
+  width: "6px",
+  height: "27px",
+  background: "white",
+});
+
+const HorizontalLine = styled("div")({
```

Comment:
The horizontal line is over the timeline track.

![Screenshot 2024-10-23 at 10 58 59 AM](https://github.com/user-attachments/assets/f28eb0f5-8fd1-4d6f-a41d-13726b29c921)

Reply:
I'm not seeing this on Chrome v130, but I applied a z-index to the dot container in commit 5a27dad. I'm going to leave this unresolved, LMK if the issue is still present and I'll try something else. 

<img width="304" alt="HR Overlap" src="https://github.com/user-attachments/assets/83b1eb22-a016-4221-a7ab-b5f10c6e6a04">

Reply:
The issue still remains, but is inconsistently showing. There seems to be a small gap between the TopConnector and BottomConnector which makes it so the border is sometimes showing. (Unsure why only sometimes) But if I zoom in 200% on Chrome and increase the border size to 5px, it becomes more noticeable. 

![Screenshot 2024-10-24 at 12 49 13 PM](https://github.com/user-attachments/assets/003de208-0575-4a29-99cd-5b738c9ca5cd)

I'm sure it is possible to accomplish a fix using box-sizing or negative margins, but I tried them and they didn't seem to work without a lot of tweaking. They would obviously be a better solution, so if you're able to figure it out, then even better. Feel free to disregard my solution below, if you find a better one! It was just my initial thoughts.

A working, but not ideal, solution I found was just setting a CSS variable for the border-width in the `StyledDialogContent` where it can be shared, then using that to calculate the connector heights:

```javascript
const StyledDialogContent = styled(DialogContent, { shouldForwardProp: (p) => p !== "headerRow" })<{
  headerRow: boolean;
}>(({ headerRow }) => ({
  "--border-bottom-width": "1px",  // I recommend using 1px instead of 0.5px, on a screen they are equivalent
  
  ...
  
}));

```

Then we can use this wherever we define our borders:

```javascript
const StyledHeaderRow = styled(Grid)({
  borderBottom: "var(--border-bottom-width) solid #375F9A",
  
  ...
  
});

const StyledEventRow = styled(Grid)({
  borderBottom: "var(--border-bottom-width) solid #375F9A",
  
  ...
  
});
```


Finally, just extend the connect height by half the border for both TopConnector and BottomConnector
```javascript
const TopConnector = styled("div")({
  height: "calc(27px + var(--border-bottom-width) / 2)",
  
  ...
  
});

const BottomConnector = styled("div")({
  height: "calc(27px + var(--border-bottom-width) / 2)",
  
  ...
  
});
```

Result:
![Screenshot 2024-10-24 at 1 26 29 PM](https://github.com/user-attachments/assets/6d62359c-280e-4282-918a-97de6579f29b)

![Screenshot 2024-10-24 at 1 24 36 PM](https://github.com/user-attachments/assets/9e8bc685-a7a7-4247-8929-ec1e9fe8bbc8)

Reply:
Thanks for the advice on using `5px`, I can see it clearly with any zoom at 5px. For some reason, even at 250% zoom on `0.5px`, the gap wasn't visible. Regardless, I like your solution and implemented it in commit 8eee7e8.

Interestingly, `0.5px` renders differently for me than `1px`, so I left it at `0.5px`. I'm assuming the reason it's rendering differently for me vs you is also the reason you can see the gap and I can't.

<details><summary>0.5px vs 1.0px</summary>
<p>

(I don't know whether this will actually show up well in a screenshot 😃)

`0.5px`

<img width="564" alt="Half-Px-Border" src="https://github.com/user-attachments/assets/41cd3828-3b2e-46d1-8693-1bcf3c004e53">

`1px`

<img width="564" alt="1-Px-Border" src="https://github.com/user-attachments/assets/e7dfedb8-0114-4b17-ae27-5eea5b56391b">

</p>
</details>